packages/core/src/utils/path.ts

```diff
@@ -66,6 +66,14 @@ const resolvePathFromProject = (...paths: string[]): string =>
  */
 const getPackagePath = () => resolvePathFromProject("package.json");
 
+/**
+ * Resolves a path to the package.json
+ *
+ * @returns A path to the package.json relative to project path
+ */
+const getPathToPlugin = (pluginName: string) =>
+  path.resolve.apply(__dirname, ["..", "..", "packages", pluginName]);
```

Comment:
this only seems relative to the monorepo not an actual react native project.....was this to fulfill the unit test? if so we need to use mocks instead of creating code based around unit test